Textile/Classes/Public/TextileApi.m

```diff
@@ -97,14 +98,24 @@ + (NSString *)initializeCreatingNewWalletAndAccount:(NSString *)repoPath debug:(
 }
 
 + (BOOL)launch:(NSString *)repoPath debug:(BOOL)debug error:(NSError * _Nullable __autoreleasing *)error {
-  RequestsHandler *requestsHandler = [[RequestsHandler alloc] init];
-  Messenger *messenger = [[Messenger alloc] init];
+  RequestsHandler *requestsHandler = [[RequestsHandler alloc] initWithTextile:Textile.instance];
+  Messenger *messenger = [[Messenger alloc] initWithTextile:Textile.instance];
+
   Textile.instance.requestsHandler = requestsHandler;
   Textile.instance.messenger = messenger;
-  MobileMobile *node = [Textile newTextile:repoPath debug:debug requestsHandler:requestsHandler messenger:messenger error:error];
+
+  MobileRunConfig *config = [[MobileRunConfig alloc] init];
+  config.repoPath = repoPath;
+  config.debug = debug;
+  config.cafeOutboxHandler = requestsHandler;
+
+  MobileMobile *node = MobileNewTextile(config, messenger, error);
   if (node) {
     Textile.instance.node = node;
     [Textile.instance createNodeDependants];
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Important change here. There are certain things we want to initialize or start on the trip around the runloop after the node is created. Examples are creating the `NSURLSession` that handles upload updates from iOS and the lifecycle manager that is responsible for starting and stopping the node.